proxy/http/Http1ClientSession.cc

```diff
@@ -456,11 +451,16 @@ Http1ClientSession::new_transaction()
 
   read_state = HCS_ACTIVE_READER;
 
-  trans.set_proxy_ssn(this);
+  ink_assert(_txn == nullptr);
+  _txn        = new Http1Transaction();
```

Comment:
I bit worry about negative performance impact on keep-alive cases. It's preferred that allocated memory of Http1Transaction is reused by using placement new or ClassAllocator.